backend/assessments/2/src/main/java/com/assessment/assessmenttwo/dao/InventoryDAO.java

```diff
@@ -0,0 +1,16 @@
+package com.assessment.assessmenttwo.dao;
+
+import com.assessment.assessmenttwo.dto.InventoryDTO;
+import com.assessment.assessmenttwo.dto.ShoppingCartDTO;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+//import java.util.int;
+@Repository
+public interface InventoryDAO extends CrudRepository<InventoryDTO, Integer> {
```

Comment:
DAO is a separate pattern and different to Repository pattern. If you want to extend Repo then name it as `InventoryRepository`